platform/windows/display_server_windows.cpp

```diff
@@ -227,7 +227,11 @@ Error DisplayServerWindows::file_dialog_show(const String &p_title, const String
 	for (const String &E : p_filters) {
 		Vector<String> tokens = E.split(";");
 		if (tokens.size() == 2) {
+#if defined(WINDOWS_ENABLED)
```

Comment:
This is always true, `DisplayServerWindows` only runs on Windows

Reply:
Why does compilation fail on Linux platform?

Reply:
Where, did it do that?

Reply:
The PR I submitted last night.

Reply:
You probably had the error in `Test Godot Project` which isn't related to this, remove the checks here and try it, it doesn't make sense that it would be run as `DisplayServerWindows` isn't run on Linux

Reply:
Sorry. It test fail on The Linux platform.

Reply:
Please link which test it failed on

Reply:
Your original PR didn't fail so there's no error there:
https://github.com/godotengine/godot/actions/runs/6264701280

Reply:
Sorry I misremembered. why it was closed

Reply:
Should I continue to resolve this issue?

Reply:
There's other alternatives that might be more complete, but you keep working on it if you feel like it's a good solution

Reply:
Thanks a lot. Contributing code has taught me a lot.